ntfs_fr.md

```diff
@@ -422,11 +423,21 @@ physical_mode_id | chaine | Requis | Identifiant du mode physique (lien vers le
 trip_property_id | chaine | Optionnel | Identifiant de la propriété accessibilité (lien vers le fichier [`trip_properties.txt`](#trip_propertiestxt-optionnel))
 dataset_id | chaine | Requis | Identifiant du jeu de données ayant fourni la circulation (lien vers le fichier [`datasets.txt`](#datasetstxt-requis)).
 geometry_id | chaine | Optionnel | Identifiant du tracé représentant la circulation (lien vers le fichier [`geometries.txt`](#geometriestxt-optionnel))
+trip_pattern_id | chaine | Optionnel | Identifiant de la mission
 
     Pour préciser si la circulation est sur réservation (tout ou partie), il faut :
         Indiquer au niveau de l'horaire (fichier [`stop_times.txt`](#stop_timestxt-requis)) si la montée et/ou la descente est à réservation
         Indiquer un commentaire (optionnel) de type TAD via les fichiers [`comments.txt`](#commentstxt-optionnel) et [`comment_links.txt`](#comment_linkstxt-optionnel)
 
+### trip_patterns.txt (optionnel)
```

Comment:
From discussions, it seems that it should be a sequence of stop-point with an identical set of:
- `pickup_type`
- `drop_off_type`
- `boarding_duration`
- `alighting_duration`

The reason behind this choice are:
- some format seems to follow this convention (NeTEx for example)
- it would be closer from Navitia needs